### ISSUE TITLE
LPUSH/RPUSH does not work with multiple arguments in tutorial

### ISSUE THREAD

**Issue by jeffbmartinez** [badboy/try.redis#42]
Note: I'm new to Redis (thanks greatly for the interactive tutorial!), so apologies ahead of time if I misinterpreted the documentation or if this is otherwise user error (and happy to hear what I'm doing wrong!).

LPUSH/RPUSH does not work with multiple arguments in tutorial.

In [tutorial](http://try.redis.io/):
```
> lpush fruits apple
(integer) 1
> lpush fruits banana cherry
(error) wrong number of arguments (given 3, expected 2)
> lrange fruits 0 -1
1) "apple"
```
In redis-server installed locally (via `latest` version of [official redis docker image](https://hub.docker.com/_/redis/) in my case, currently version 3.0.7):
```
172.17.0.2:6379> lpush fruits apple
(integer) 1
172.17.0.2:6379> lpush fruits banana cherry
(integer) 3
172.17.0.2:6379> lrange fruits 0 -1
1) "cherry"
2) "banana"
3) "apple"
```

My `info` command reports:
```
172.17.0.2:6379> info
# Server
redis_version:3.0.7
...
```
Link to [the LPUSH documentation page](http://redis.io/commands/lpush) where we can see under (the history section)[http://redis.io/commands/lpush#history] it says

> History
> * \>= 2.4: Accepts multiple value arguments. In Redis versions older than 2.4 it was possible to push a single value per command.

The tutorial's `info` command:

```
> info
# Server
redis_version:3.1.999
...
```

So it seems like this should work.

P.S. Thanks for the great tutorial! Amazing way to learn about redis on the quick! Truly appreciate it

**Comment by badboy** [comment]
Thanks for the kind words. I'm happy the tutorial helps.

The reported issue looks like a bug, I'll look into it.

**Comment by badboy** [comment]
Fixed in f141f2f892702c11aa8c5632494f9234a090dbdd and deployed just a minute ago
